contracts/extensions/MultisigPermissions.sol

```diff
@@ -0,0 +1,552 @@
+
+  // Storage
+
+  IColonyNetwork colonyNetwork;
+
+  uint256 globalThreshold;
+  mapping(address => mapping(uint256 => bytes32)) internal userDomainRoles;
+
+  uint256 motionCount;
+  mapping (uint256 => Motion) motions;
+
+  // Motion Id => User => approvals
+  mapping(uint256 => mapping(address => bytes32)) motionApprovals;
+  // Motion Id => Permission => Approval Count
+  mapping(uint256 => mapping(ColonyRole => uint256)) motionRoleApprovalCount;
+
+  // Domain Skill Id => Role => Usercount
+  mapping(uint256 => mapping(uint8 => uint256)) domainSkillRoleCounts;
+
+  // Domain Skill Id => fixed threshold
+  mapping(uint256 => uint256) domainSkillThreshold;
+
+  // Overrides
+
+  /// @notice Returns the identifier of the extension
+  /// @return _identifier The extension's identifier
+  function identifier() public override pure returns (bytes32 _identifier) {
+    return keccak256("MultisigPermissions");
+  }
+
+  /// @notice Returns the version of the extension
+  /// @return _version The extension's version number
+  function version() public override pure returns (uint256 _version) {
+    return 1;
+  }
+
+  /// @notice Configures the extension
+  /// @param _colony The colony in which the extension holds permissions
+  function install(address _colony) public override auth {
+    require(address(colony) == address(0x0), "extension-already-installed");
+    colony = IColony(_colony);
+    colonyNetwork = IColonyNetwork(colony.getColonyNetwork());
+  }
+
+  /// @notice Called when upgrading the extension
+  function finishUpgrade() public override auth {}
+
+  /// @notice Called when deprecating (or undeprecating) the extension
+  /// @param _deprecated Indicates whether the extension should be deprecated or undeprecated
+  function deprecate(bool _deprecated) public override auth {
+    deprecated = _deprecated;
+  }
+
+  function getCapabilityRoles(bytes4 _sig) public pure override returns (bytes32) {
+    if (_sig == SET_USER_ROLES) {
+      return ONLY_ROOT_ROLE_MASK | ONLY_ARCHITECTURE_ROLE_MASK;
+    }
+    return bytes32(0);
+  }
+
+
+  /// @notice Called when uninstalling the extension
+  function uninstall() public override auth {
+    selfdestruct(payable(address(colony)));
+  }
+
+  // Public
+
+  modifier notExecuted(uint256 _motionId) {
+    require(!motions[_motionId].executed, "multisig-motion-already-executed");
+    _;
+  }
+
+  modifier onlyCoreRoot() {
+    require(colony.hasUserRole(msgSender(), 1, ColonyDataTypes.ColonyRole.Root), "multisig-permissions-not-core-root");
+    _;
+  }
+
+  function initialise(uint256 _globalThreshold) public onlyCoreRoot {
+    globalThreshold = _globalThreshold;
+
+    emit ExtensionInitialised();
+  }
+
+  function getGlobalThreshold() public view returns (uint256) {
+    return globalThreshold;
+  }
+
+  function createMotion(uint256 _permissionDomainId, uint256 _childSkillIndex, address[] memory _targets, bytes[] memory _data) notDeprecated public {
```

Comment:
I think this is required for a lot of the functionality that is in mind. To give an example relating to something we're in the process of adding: if I want to edit an expenditure and also make it a staged expenditure, then I would need this. I think it is more likely that this will be added in a future version of motions, rather than being removed here.